common/changes/@itwin/frontend-devtools/WZ-move-scheduleScriptReference-internal_2025-02-04-20-16.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/frontend-devtools",
+      "comment": "Moved scheduleSriptReference internal",
```

```suggestion
      "comment": "",
```

Just reacting to change in core-frontend, no need for change log.